workers/unity/Packages/com.improbable.gdk.core/Config/LocatorConfig.cs

```diff
@@ -55,16 +74,51 @@ public void SetProjectName(string projectName)
         }
 
         /// <summary>
-        ///     Sets the login token for the Locator to use.
+        ///     Sets the LocatorCredentialsType to LoginToken and sets the login token for the Locator to use.
```

Comment:
Alternative:

Sets the login token for the locator to use. Additionally, the CredentialsType will be set to LocatorCredentialsType.LoginToken.

If not:
Perhaps `CredentialsType` instead of `LocatorCredentialsType`?

Reply:
I've reworded the description according to your suggestion. :)